### ISSUE TITLE
Merge the metagenotype-dev branch into master

### ISSUE THREAD

**Issue by kimrutherford** [pombase/canto#1738]
I'd like to do this in the next two or three weeks, maybe Feb 8th.  We have a new project starting in mid-Feb, collaborating with FlyBase.  It's going require changes to Canto and it will be easier if the master branch contains the metagenotype changes.

To start the ball rolling, I'll do some testing this week using the metagenotype-dev branch in pombe/single-organism mode.  I'll make issues for any problems I find.

We should have a chat about how development should work after we merge the current changes into the master branch.  Should we continue to develop the meta-genotype stuff on a branch?

**Comment by kimrutherford** [comment]
Hi James.

Could you try Canto in single organism mode when you get a chance?  I plan to try it myself tomorrow.

**Comment by kimrutherford** [comment]
> Perhaps a copy of the data directory with the correct ontologies already loaded?

That's good idea.  I'll create a clean new single organism Canto instance and send you the config and data directory.

**Comment by kimrutherford** [comment]
> I'll create a clean new single organism Canto instance

I've made a clean Canto instance with the commands below.  Here's the archive of the `canto-space` directory: http://oliver1.sysbiol.cam.ac.uk/kmr44/pombe-test-canto-space-2019-02-04.tar.xz

```
./canto/script/canto_start_docker --initialise /data

# edit canto_deploy.yaml and add extension_conf_files

./canto/script/canto_docker ./script/canto_add.pl --person "Admin User" adminuser@pombase.org secret_password 0000-0001-5000-0007 admin
./canto/script/canto_docker ./script/canto_add.pl --organism "Schizosaccharomyces pombe" 4896 pombe
(cd import_export/; wget ftp://ftp.pombase.org/pombe/names_and_identifiers/gene_IDs_names.tsv)
./canto/script/canto_docker ./script/canto_load.pl --genes /import_export/gene_IDs_names.tsv --for-taxon 4896

for i in FYPO_A_E_config GO_BP_A_E_config GO_CC_A_E_config GO_MF_A_E_config PSI-MOD_A_E_config
do
(cd import_export/; wget https://curation.pombase.org/dumps/latest_build/pombe-embl/supporting_files/annotation_ex_config/$i)
done

./canto/script/canto_docker ./script/canto_load.pl --process-extension-config --ontology https://curation.pombase.org/dumps/latest_build/ontologies/go-basic.obo --ontology https://curation.pombase.org/dumps/latest_build/ontologies/fypo-simple.obo --ontology https://raw.githubusercontent.com/The-Sequence-Ontology/SO-Ontologies/master/so-simple.obo --ontology https://raw.githubusercontent.com/pombase/fypo/master/peco.obo --ontology https://curation.pombase.org/dumps/latest_build/pombe-embl/mini-ontologies/fypo_extension.obo --ontology https://curation.pombase.org/dumps/latest_build/pombe-embl/mini-ontologies/has_qualifier_range.obo --ontology https://curation.pombase.org/dumps/latest_build/pombe-embl/mini-ontologies/pombase_gene_expression_ontology.obo --ontology https://raw.githubusercontent.com/pato-ontology/pato/master/pato.obo 
```

This is the `canto_deploy.yaml`:

```
---
Model::TrackModel:
  connect_info:
    - dbi:SQLite:dbname=/data/track.sqlite3
  schema_class: Canto::TrackDB
data_directory: /data


extension_conf_files:
  - /import_export/FYPO_A_E_config
  - /import_export/GO_BP_A_E_config
  - /import_export/GO_CC_A_E_config
  - /import_export/GO_MF_A_E_config
  - /import_export/PSI-MOD_A_E_config

```

I think that's all that's needed for testing.

**Comment by kimrutherford** [comment]
I mis-configured my single organism mode Canto.  I forgot:

```
 instance_organism:
   taxonid: 4896
```

Without the ` instance_organism` it defaults to multi-organism mode.  I forgot that.

With that change a lot of the issues I found disappear.

**Comment by kimrutherford** [comment]
Here's a new `canto-space` archive with the `instance_organism` fix:
http://oliver1.sysbiol.cam.ac.uk/kmr44/pombe-test-canto-space-2019-02-07.tar.xz

**Comment by kimrutherford** [comment]
I've merged the PHI changes into the master branch and pushed it to GitHub.  I've committed a small fix for multi-organism mode (https://github.com/pombase/canto/commit/a4b5ecef19977e359eaeb8351056e3a3e283d868) but otherwise it's all working find as far as I can tell.

I've also merged a small branch with changes for FlyBase, mostly adding a way to create new sessions from a JSON file of PubMed IDs and genes.  See #1779 for details.
